<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">letter space:100px</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDraw"/>
   <addaction name="actionWireFrame"/>
   <addaction name="actionClear"/>
   <addaction name="actionloadModel"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionDraw">
   <property name="text">
    <string>绘制</string>
   </property>
  </action>
  <action name="actionWireFrame">
   <property name="text">
    <string>线框</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>清空</string>
   </property>
  </action>
  <action name="actionloadModel">
   <property name="text">
    <string>loadModel</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
